MsWheaPkg/MsWheaReport/MsWheaReportHER.c

```diff
@@ -429,6 +429,12 @@ MsWheaClearAllEntries (
 
   NameSize = sizeof (CHAR16);
   Name     = AllocateZeroPool (NameSize);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (Name == NULL) {
+    return EFI_OUT_OF_RESOURCES;

```

Comment:
can you update the funciton header to reference EFI_OUT_OF_RESOURCES?

Reply:
Updated